budget-binder-multiplatform-app/src/commonMain/kotlin/de/hsfl/budgetBinder/presentation/Screen.kt

```diff
@@ -6,6 +6,7 @@ sealed class Screen {
     object Register : Screen()
     object Dashboard : Screen()
     object Settings : Screen()
+    object SettingsChangeUserData : Screen()
```

Comment:
Wozu ist der?

Reply:
Den hat @Creditore eingebaut als Screen, in der das ChangeUser Formular ist. so kommt man von Settings.onChangeUserDataButton() -> Screen.SettingsChangeUserData.